packages/core-bridge/src/runtime.rs

```diff
@@ -219,6 +219,46 @@ pub fn start_bridge_loop(
                     client.get_client().set_api_key(Some(key));
                     send_result(channel.clone(), callback, |cx| Ok(cx.undefined()));
                 }
+                RuntimeRequest::SendClientRequest { client, call, callback } => {
+                    core_runtime.tokio_handle().spawn(async move {
+                        match client_invoke((*client).clone(), call).await {
+                            Ok(bytes) => send_result(channel.clone(), callback, move |cx| {
+                                JsArrayBuffer::from_slice(cx, &bytes)
+                            }),
+                            Err(err) => send_error(channel.clone(), callback, |cx| {
+                                match err {
+                                    RpcError::Tonic(err) => {
+                                        let code = cx.number(err.code() as u32);
+                                        let metadata = JsObject::new(cx);
+                                        let details = err.details();
+                                        if !details.is_empty() {
+                                            let k = cx.string("grpc-status-details-bin");
```

Comment:
Have you confirmed that these `grpc-status-details-bin` ends up being decoded properly by the Client, i.e. same as those coming from a JS Connection?

Reply:
Yup, there's a test that verifies that.